earthpy/tests/test_spatial.py

```diff
@@ -1,6 +1,32 @@
-from earthpy import spatial
+import earthpy.spatial as es
+from shapely.geometry import Polygon, Point
+import geopandas as gpd
+import pandas as pd
+import pytest
 
 
-def test_dummy():
-    """"Dummy test that will pass."""
-    assert 1 == 1
+def test_extent_to_json():
+    """"Unit tests for extent_to_json()."""
+    # giving a list [minx, miny, maxx, maxy] makes a polygon
```

Comment:
We've been following rasterio and PEP 8 suggests capitalizing the first word of each comment line. Do you mind making that update unless you have a strong argument against it? Not all packages do it but i've been following Sean's lead as he seems to really know what he is doing and i appreciate how robust that package is.

Reply:
Bah oops. Fixed with 09a5dbaf9f50179859b5bab53cae4ce6b396a156